src/main/sass/wasteland-ventures/chat-message/system-message/weapon-attack.sass

```diff
@@ -1,6 +1,6 @@
 @use "../../../constants.sass" as constants
 
-.wasteland-ventures.chat-message.system-message.weapon-attack
+.wasteland-ventures.chat-message.system-message.weapon-attack, .wasteland-ventures.chat-message.system-message.general-roll
```

Comment:
I think we shouldn't just lump this together in a file called "weapon-attack.sass". Common stuff should be put in a common class and that then put on both chat cards.